.github/workflows/test-sui.yaml

```
@@ -33,8 +33,8 @@ jobs:
       - name: Spin up Sui Network
         run: nohup sh -c "sui-test-validator" > nohup.out 2> nohup.err < /dev/null &
 
-      - name: Sleep for 30 seconds
-        run: sleep 30s
+      - name: Sleep for 40 seconds
```

```suggestion
      - name: Wait for Sui network spinup
```
Is there a certain log that can we can wait for to check if the network is ready instead of just sleeping?

---

fixed by using https://github.com/nev7n/wait_for_response/tree/master which simply does http head call periodically to the given endpoint which is the faucet endpoint here.